Makefile

```diff
@@ -39,6 +39,7 @@ check:
 	tests/pylint/runpylint.py
 
 test:
+	@which nosetests || (echo "*** Please install python-nose ***"; exit 2)
```

Comment:
In this case we'd need to change the messages for coverage and po-pull targets as well.